src/components/chimera1.svg

```diff
@@ -0,0 +1 @@
632813 1.738281-5.992188 1.941406-8.816407.53125-20.796874-10.40625-30.359374-6.894532-32.492187-3.585938-1.445313 2.917969-4.425781 4.757812-7.683594 4.742188-3.117187.019531-6.015625-1.597657-7.640625-4.257813s-1.742187-5.980469-.300781-8.746094c4.558594-7.472656 12.558594-12.160156 21.308594-12.488281.210937-.125.351562-.320312.574219-.425781 5.40625-2.707031 10.542968-5.929688 15.332031-9.621094h-37.859375c-19.742188 16.929687-44.84375 26.316406-70.851563 26.480469-13.390625-.554688-26.226562-5.5-36.527343-14.070313-5.164063 27.371094-7.976563 55.132813-8.40625 82.980469v1.007812c.042968 21.375 15.144531 39.761719 36.105468 43.960938v-34.429688c0-4.875 3.953125-8.828124 8.828125-8.828124s8.828125 3.953124 8.828125 8.828124v97.105469c-.023437 4.824219 1.945313 9.449219 5.445313 12.773438 3.542969 3.398437 8.339843 5.171875 13.242187 4.890625 9.652344-1.082032 16.871094-9.371094 16.621094-19.078125v-25.070313c0-19.5 15.808594-35.308594 35.3125-35.308594h123.585938c6.214843.035157 12.3125 1.730469 17.65625 4.90625v-40.21875c0-4.875 3.949218-8.828124 8.824218-8.828124s8.828125 3.953124 8.828125 8.828124v97.105469c-.007812 4.859375 1.992188 9.511719 5.523438 12.851563 3.53125 3.34375 8.285156 5.078125 13.140625 4.800781 9.65625-1.066406 16.890625-9.351563 16.648437-19.066406v-104.519531c-.0625-53.601563-43.503906-97.039063-97.105469-97.101563zm0 0"/><path d="m88.757812 494.269531c0 9.75 7.902344 17.652344 17.652344 17.652344 9.753906 0 17.65625-7.902344 17.65625-17.652344v-44.757812c-13.371094-1.910157-25.757812-8.105469-35.308594-17.65625zm0 0"/><path d="m335.929688 450.128906h-17.65625v44.140625c0 9.75 7.90625 17.652344 17.65625 17.652344s17.65625-7.902344 17.65625-17.652344v-26.484375c0-9.75-7.90625-17.65625-17.65625-17.65625zm0 0"/><path d="m124.066406 194.128906c0-33.339844-13.242187-41.398437-39.582031-57.457031-1.386719-.839844-2.878906-1.765625-4.308594-2.648437-6.457031 8.296874-12.4375 16.960937-17.902343 25.945312-3.019532 5.28125-6.480469 10.300781-10.34375 15.003906-6.609376 6.640625-15.597657 10.359375-24.964844 10.328125-14.628906 0-26.484375 11.859375-26.484375 26.484375s11.855469 26.484375 26.484375 26.484375h52.964844c30.523437 0 44.136718-6.375 44.136718-44.140625zm-35.308594-17.652344h-8.828124c-4.875 0-8.828126-3.953124-8.828126-8.828124s3.953126-8.828126 8.828126-8.828126h8.828124c4.875 0 8.828126 3.953126 8.828126 8.828126s-3.953126 8.828124-8.828126 8.828124zm0 0"/><path d="m358.085938 16.34375c-75.476563 45.285156-83.066407 125.878906-83.863282 140.269531 19.953125 35.222657 25.425782 67.796875 16.066406 97.015625-4.558593 14.039063-12.1875 26.886719-22.332031 37.605469h-26.308593c14.824218-10.71875 25.957031-25.769531 31.871093-43.078125 7.855469-24.808594 2.558594-53.40625-15.804687-84.921875-.804688-1.332031-1.234375-2.859375-1.234375-4.414063 2.5625-64.730468 37.3125-123.898437 92.601562-157.660156 4.183594-2.386718 9.507813-.976562 11.964844 3.167969 2.453125 4.140625 1.136719 9.488281-2.960937 12.015625zm0 0"/></svg>
```

Comment:
🔍hard to read :)

Reply:
who cares it's an svg.  it's only purpose is too look nice no? ;)